### PR TITLE
[Exporter] add fix for prometheus exporter build

### PR DESCRIPTION
**Pull request by rochaudhari**
Signed-off-by: Roshan Chaudhari rochaudhari@nvidia.com

Fixes https://github.com/open-telemetry/opentelemetry-cpp/issues/1728

For prometheus exporter, required dependencies are not loaded in repository.bzl. When Opentelemetry is imported in some other project, these dependencies are missing.

### CODE REVIEW

File: bazel/extra_deps.bzl [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
```
Comment (lalitb) [inline bazel/extra_deps.bzl:3]: @rochaudhari   As per the[ CNCF guidelines](https://github.com/cncf/foundation/blob/master/copyright-notices.md), the copyright header should be - " Copyright The OpenTelemetry Authors", as we have it all other files within this repo. These two lines for copyright and license should be enough:

```
// Copyright The OpenTelemetry Authors
// SPDX-License-Identifier: Apache-2.0
```

**Comment by lalitb** [comment]
As of now, CI build is failing for #1791, this PR can be merged once the build is fixed.

**Review by marcalff — APPROVED** [review]
LGTM, thanks for the fix.

File: INSTALL.md [hunk 2]
```diff
@@ -229,6 +229,11 @@ load("@io_opentelemetry_cpp//bazel:repository.bzl", "opentelemetry_cpp_deps")
 
 opentelemetry_cpp_deps()
 
+# Load extra dependencies required for openTelemetry
```
Comment (ThomsonTan) [inline INSTALL.md:232]: nit:
```suggestion
# Load extra dependencies required for OpenTelemetry
```

**Review by lalitb — APPROVED** [review]

File: INSTALL.md (hunk 2 above)
Comment (rochaudhari) [reply INSTALL.md]: Done!

**Comment by lalitb** [comment]
@rochaudhari  -  CI test is failing for code formatting. Can you fix the formatting as suggested here - https://github.com/open-telemetry/opentelemetry-cpp/blob/main/CONTRIBUTING.md#how-to-send-pull-requests

**Comment by rochaudhari** [comment]
> @rochaudhari - CI test is failing for code formatting. Can you fix the formatting as suggested here - https://github.com/open-telemetry/opentelemetry-cpp/blob/main/CONTRIBUTING.md#how-to-send-pull-requests

Strange! After running format.sh as well, the file looks same. I updated the patch again.

**Comment by rochaudhari** [comment]
@lalitb Can we merge this?
When is the next release planned?

**Comment by esigo** [comment]
> @lalitb Can we merge this? When is the next release planned?

@rochaudhari, I'll prepare the release this weekend.

**Pull request merged**